test.js

```diff
@@ -0,0 +1,6 @@
+import test from 'ava';
+import Ten from './ten';
+
+test(t => {
+    t.is(Ten / Ten, 1);
```

Comment:
Good test. I think multiplication could also be a concern, what about

```javascript
t.is(Ten * 1, Ten);
```

or such?